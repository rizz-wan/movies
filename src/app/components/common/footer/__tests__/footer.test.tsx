import { Footer } from "..";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

describe("Footer tests", () => {
  describe("Snapshot tests", () => {
    it("Should render footer and match snapshot", () => {
      const result = shallow(<Footer />);
      expect(result).toBeTruthy();
      expect(result).toMatchSnapshot();
    });
  });
});
