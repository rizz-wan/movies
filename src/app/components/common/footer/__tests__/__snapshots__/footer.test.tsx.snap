// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer tests Snapshot tests Should render footer and match snapshot 1`] = `ShallowWrapper {}`;
